R/get_input_files.R

```diff
@@ -71,7 +71,8 @@ get_input_files <- function(portfolio_name_ref_all) {
   input_file_path <- path(input_path, paste0(portfolio_name_ref_all, ".csv"))
 
   portfolio <- pacta.portfolio.import::read_portfolio_csv(input_file_path)
-  portfolio <- portfolio %>% select(-"investor_name", -"portfolio_name")
+  portfolio <- portfolio %>%
+    select(-dplyr::starts_with(c("investor_name", "portfolio_name")))
```

Comment:
```suggestion
    select(-dplyr::any_of(c("investor_name", "portfolio_name")))
```
I'd rather use `dplyr::any_of()` here... it's more to the point.

Reply:
Happy with that!